src/Arcus.Template.WebApi/Dockerfile

```
@@ -0,0 +1,23 @@
+FROM mcr.microsoft.com/dotnet/core/aspnet:2.2-stretch-slim AS base
+WORKDIR /app
+EXPOSE 80
+
+FROM mcr.microsoft.com/dotnet/core/sdk:2.2-stretch AS build
+WORKDIR /src
+COPY ["Arcus.Template.WebApi/Arcus.Template.WebApi.csproj", "Arcus.Template.WebApi/"]
+RUN dotnet restore "Arcus.Template.WebApi/Arcus.Template.WebApi.csproj"
+COPY . .
+WORKDIR "/src/Arcus.Template.WebApi"
+RUN dotnet build "Arcus.Template.WebApi.csproj" -c Release
+RUN dotnet new -i "bin/Release/Arcus.Template.WebApi.*.nupkg"
+RUN dotnet new arcus-webapi -n MyProject -o /output
+WORKDIR "/output"
+RUN dotnet build -c Release -o /app	
```

We can actually skip building as publish builds as well
```suggestion
RUN dotnet publish "MyProject.csproj" -c Release -o /app
```

---

Will publish not only build the app when there is source code changes?

---

Either retore+publish or build+publish

---

Publish does everything that it should to output the artifact so that should be good to go

---

Yeah I thought so too, but apperently with only `publish` in the Dockerfile, the build fails: non `.dll` file found.

The restore is run though (with `publish`) but no building of the source happen.

I checked this locally on a fresh project; and there I also got the same result: `publish` does `restore` but no `build`.

---

Resolving as you are correct